src/index.html

```diff
@@ -4,13 +4,27 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="search-form search-form--big">
+      <label class="search-form__label search-form__label--big">
+        <input data-qa="keypress"
+          type="search"
+          class="search-form__input search-form__input--big"
+          placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
```suggestion
        <input 
          data-qa="keypress"
          type="search"
          class="search-form__input search-form__input--big"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
Pay attention to the formatting